cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -1642,6 +1642,107 @@ const char* convert_toUTF8(int64_t context, const char* value, int32_t value_len
   return value;
 }
 
+// Calculate the levenshtein distance between two string values
+FORCE_INLINE
+gdv_int32 levenshtein_utf8_utf8(int64_t context, const char* in1, int32_t in1_len,
```

Comment:
I dont think this algorithm is written to work for utf-8 input. utf-8 uses multiple bytes to encode a character... this algorithm doesn't handle this correctly..

Can you change the name of the function to not include utf-8 in the name?

Reply:
Yeah, of corse!
I go change this now

Reply:
@vvellanki 
I just made a new push with the changes, thanks!